tests/test_eclmateriallawmanager.cpp

```diff
@@ -236,7 +236,7 @@ int main()
 
     {
         typedef Opm::EclMaterialLawManager<MaterialTraits> MaterialLawManager;
-        typedef typename MaterialLawManager::MaterialLaw MaterialLaw;
+        typedef MaterialLawManager::MaterialLaw MaterialLaw;
```

Comment:
> I would think the typename here would be disallowed anyway since it is not in a template context, any idea why it is accepted by the new compilers? I don't even get a warning.

I'm pretty sure that newer compilers accept this because the final version of C++ 2011 allows this even in non-template contexts.